Form-Controls/index.html

```diff
@@ -13,15 +13,42 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <form action="#" method="post">
+      <h2>T-Shirt Order Form</h2>
+        
+      <label for="name">Customer Name</label>
+      <input type="text" id="name" name="name" placeholder="Enter your name" minlength="2" required pattern="[A-Za-z\s]{2,}">
```

Comment:
What would happen if the customer had a name like O'Donnell?
https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/